app/grandchallenge/workstations/urls.py

```diff
@@ -29,11 +29,21 @@
         SessionCreate.as_view(),
         name="workstation-session-create",
     ),
+    path(
+        "<slug>/sessions/create/<str:workstation_path>/<uuid:pk>/",
```

Comment:
This should be able to handle any changes to the CIRRUS URLS:

```suggestion
        "<slug>/sessions/create/<path:workstation_path>",
```